packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -1,114 +1,144 @@
-   * @defaultValue '0'
+   * Border radius for the x-axis of a rounded rect.
+   * @defaultValue 0
    */
-  borderRadius?: number;
+  borderRadiusX?: number;
+
   /**
-   * x-coordinate
-   * @defaultValue '0'
+   * Border radius for the y-axis of a rounded rect.
+   * @defaultValue 0
    */
-  xPos?: number;
+  borderRadiusY?: number;
+
   /**
-   * y-coordinate
-   * @defaultValue '0'
+   * Note: x and y should be optional properties [or removed], with relative positioning being the default [or only] positioning mechanism.
+   * Note: The behavior of element intersections is undefined.
    */
-  yPos?: number;
-}
 
-export interface ShimmerTokens {
   /**
-   * Specifies the distance traveled from starting position.
-   * By default, the animation will animate from 0 to 30 over the course of 7000ms, since speed = distance / time, so
-   * speed ≈ 0.004 (higher the speed, faster it moves).
-   * @defaultValue '30'
+   * The x-axis position of the rect element's top-left corner in the Shimmer relative to the origin [top-left, (0,0)].
+   */
+  x: number;
+
+  /**
+   * The y-axis position of the rect element's top-left corner in the Shimmer relative to the origin [top-left, (0,0)].
    */
-  toValue?: number;
+  y: number;
+}
+
+export interface ShimmerTokens extends IBackgroundColorTokens {
   /**
-   * Specifies the time required to travel a given distance in milliseconds
-   *
-   * @defaultValue '7000'
+   * Specifies the Shimmer effect angle in degrees (produced by a gradient)
+   * @defaultValue 0
    */
-  duration?: number;
+  angle?: number;
+
   /**
    * Specifies the animation delay time in milliseconds
-   * @defaultValue '0'
+   * @defaultValue 0
    */
   delay?: number;
+
   /**
-   * Gradient tint color
-   * @defaultValue 'white'
-   */
-  gradientTintColor?: ColorValue;
-  /**
-   * Shimmer element tint color, no-op for image based shimmer
-   * @defaultValue 'E1E1E1' for light mode, '404040' for dark mode
+   * Specifies the time required to traverse the control in milliseconds
+   *
+   * @defaultValue 2000
    */
-  shimmerTintColor?: ColorValue;
+  duration?: number;
+
   /**
-   * Width of the shimmer view
-   * @defaultValue '200'
+   * Specifies the gradient opacity.
+   * @defaultValue 1
    */
-  width?: number;
+  gradientOpacity?: number;
+
   /**
-   * Height of the shimmer view
-   * @defaultValue '100'
+   * Color you see when the shimmer wave is not animating.
+   * @defaultValue theme.colors.bodyFrameDivider
    */
-  height?: number;
+  shimmerColor?: ColorValue;
+
   /**
-   * Specifies the gradient angle, value should be anywhere between -1 and 1
-   * -1 to 1 maps to 90 degrees horizontally to ≈ 120 degrees
-   * @defaultValue '0'
+   * Defines the tip color of the wave which has a linear gradient.
+   * from shimmerColor to shimmerWaveColor to shimmerColor.
+   * @defaultValue '#E1E1E1'
    */
-  angle?: number;
+  shimmerWaveColor?: ColorValue;
+
```

Comment:
Juuuust noticed. These cannot be color values because rn-svg doesn't support ColorValue. This has to be `string` or `string | number`

Reply:
Is this being used in the svg itself? I thought it was the gradient, is that an svg?